engine/src/main/java/org/terasology/world/block/tiles/WorldAtlasImpl.java

```diff
@@ -250,9 +269,18 @@ private void createTextureAtlas(final Texture texture) {
     }
 
     private ByteBuffer[] createAtlasMipmaps(int numMipMaps, Color initialColor, List<BlockTile> tileImages, String screenshotName) {
+        return createAtlasMipmaps(numMipMaps, initialColor, tileImages, screenshotName, Lists.newArrayList());
+    }
+
+    private ByteBuffer[] createAtlasMipmaps(int numMipMaps, Color initialColor, List<BlockTile> tileImages, String screenshotName, List<BlockTile> alphaMaskTiles) {
         ByteBuffer[] data = new ByteBuffer[numMipMaps];
         for (int i = 0; i < numMipMaps; ++i) {
             BufferedImage image = generateAtlas(i, tileImages, initialColor);
+            if (alphaMaskTiles.size() > 0) {
+                BufferedImage alphaMask = generateAtlas(i, alphaMaskTiles, Color.BLACK);
+                storeGreyscaleMapIntoAlpha(image, alphaMask);
```

Comment:
Hrm.  Going over this again...  I suppose it feels right to keep it generic is because of how the entry method is set up at the moment.  createAtlasMipmaps(...) is shared between the color, height, and normal atlas textures.  So it would be odd to have the variables named very specifically instide of createAtlasMipmaps when it has come in through more generic means.